packages/discovery-react-components/src/components/CIDocument/components/CIDocument/CIDocument.tsx

```diff
@@ -269,6 +271,7 @@ const CIDocument: FC<CIDocumentProps> = ({
       if (hasRelation(highlightedList)) {
         setActiveIds(highlightedList[0].allAttributeIds);
       } else {
+        setClickedItemType('');
```

Comment:
I'm concerned here that we had to add `setClickedItemType('')` in multiple places. It seems our code isn't structured well if that was required.

What is the actual intent here? Are we trying to reset something while also setting something else? If so, could we do something like:

```
resetAll();
setThatOneThing();
```